src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -41,6 +41,88 @@ public static FileInfo GetGuidNameForNewFile(string test, string extension)
         }
 
         [Fact]
+        public void T201_FileFormatException()
+        {
+            var e = new FileFormatException();
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T202_FileFormatException()
+        {
+            var e2 = new IOException("Test");
+            var e = new FileFormatException("Test", e2);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T203_FileFormatException()
+        {
+            var partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            var e = new FileFormatException(partUri);
+            Assert.NotNull(e);
```

Comment:
Ditto, e.g.
```C#
Assert.Same(partUri, e.SourceUri);
```
Etc.  I'll stop commenting on these, but it applies to the other tests below.